### PR TITLE
2.0 preview

### PR DESCRIPTION
**Pull request by notwaldorf**
Straight forward port

### CODE REVIEW

**Comment by notwaldorf** [comment]
@bicknellr this is ready for 👀

File: bower.json [hunk 1]
```diff
@@ -22,14 +22,28 @@
...
+    "paper-styles": "polymerelements/paper-styles#2.0-preview"
   },
   "devDependencies": {
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0",
-    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#^1.0.0",
-    "web-component-tester": "^4.0.0",
-    "test-fixture": "polymerelements/test-fixture#^1.0.0",
-    "iron-component-page": "PolymerElements/iron-component-page#^1.0.0"
+    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#2.0-preview",
+    "test-fixture": "PolymerElements/test-fixture#^3.0.0-rc.1",
```
Comment (bicknellr) [inline bower.json]: WCT 4 & 6 reference the right versions of test-fixture.

File: bower.json [hunk 2]
```diff
@@ -22,14 +22,28 @@
...
+    "web-component-tester": "Polymer/web-component-tester#^6.0.0-prerelease.6",
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^1.0.0-rc.1"
```
Comment (bicknellr) [inline bower.json]: `devDependencies` seems to have lost `iron-component-page` even though it's still used in `index.html`.

**Comment by notwaldorf** [comment]
@bicknellr oops, done!

File: paper-material.html [hunk 3]
```diff
@@ -33,14 +33,14 @@
     <style include="paper-material-shared-styles"></style>
     <style>
       :host([animated]) {
-        @apply(--shadow-transition);
+        @apply --shadow-transition;
       }
       :host {
         @apply(--paper-material);
```
Comment (bicknellr) [inline paper-material.html]: Here's another `@apply(x);` -> `@apply x;`

Comment (notwaldorf) [inline paper-material.html]: done!

**Comment by bicknellr** [comment]
WRT [this comment], is this a breaking change introduced in webcomponentsjs? Or did this just never work?

**Pull request merged**